### ISSUE TITLE
Propagate Ctrl+C

### ISSUE THREAD

**Issue by mikaelmello** [mikaelmello/inquire#154]
Inquire captures Ctrl+C, maybe it should propagate

**Comment by mikaelmello** [comment]
Certainly. I'm thinking though that it should propagate by default (a breaking change), and then if they want to handle it themselves, they can opt-in. Wdyt?

**Comment by alxpettit** [comment]
Ctrl+C isn't even universal on all terminal configurations. Would really like a way to make my program not close when hitting it.
